lib/iris/fileformats/grib/_grib_cf_map.py

```diff
@@ -161,6 +164,7 @@
 
 CF_TO_GRIB2 = {
     CFName(None, 'cloud_area_fraction_assuming_maximum_random_overlap', '1'): G2Param(2, 0, 6, 1),
+    CFName(None, 'cloud_area_fraction_assuming_maximum_random_overlap', '1'): G2Param(2, 0, 6, 1),
```

Comment:
> This line now appears twice!

hmm, this is supposed to be protected by validation rules.  i'm chasing the reason why this was missed.

in the meantime, here's an automated update